<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTaskEditForm</class>
 <widget class="QDialog" name="CTaskEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit task</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblTask">
         <property name="text">
          <string>Task:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbxTaskComplete">
         <property name="text">
          <string>Task complete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="teTask"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblDueDateTime">
       <property name="text">
        <string>Due date/time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="dteDueDateTime">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnSetDueDateTime">
       <property name="text">
        <string>Set/clear due</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblAlertDateTime">
       <property name="text">
        <string>Alert date/time:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateTimeEdit" name="dteAlertDateTime">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnSetAlertDateTime">
       <property name="text">
        <string>Set/clear alert</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblPriority">
       <property name="text">
        <string>Priority:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cboPriority"/>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblProjects">
         <property name="text">
          <string>Project(s):</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listProjects">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnNewProject">
         <property name="text">
          <string>New project...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lblNotes">
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="teNotes"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbxOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbxOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>CTaskEditForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbxOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>CTaskEditForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetAlertDateTime</sender>
   <signal>clicked()</signal>
   <receiver>CTaskEditForm</receiver>
   <slot>showHideAlertDateTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSetDueDateTime</sender>
   <signal>clicked()</signal>
   <receiver>CTaskEditForm</receiver>
   <slot>showHideDueDateTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showHideDueDateTime()</slot>
  <slot>showHideAlertDateTime()</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
